pkgs/build-support/fetchpijul/builder.sh

```diff
@@ -0,0 +1,10 @@
+if [ -e .attrs.sh ]; then source .attrs.sh; fi
+source $stdenv/setup
+
+pijul clone \
+  ${change:+--change "$change"} \
+  ${state:+--state "$state"} \
+  --channel "$channel" \
+  "$url" "$out"
+
+rm -rf "$out/.pijul"
```

Comment:
We don't need to fix anything else?

Reply:
`.pijul` contains the repository metadata etc which on top of containing timestamps and whatnot also contains the order in which the patches were applied which isn't necessarily consistent. Everything else is just what the user has in the repository which shouldn't need fixing

Reply:
Please add a comment regarding why `leaveDotPijul` isn't possible at the moment to help clarify for any other reviewers.